.github/ISSUE_TEMPLATE/issue_template.yml

```
@@ -0,0 +1,47 @@
+---
+name: Issue Template
+description: Report a problem or bug related to FZF to help us improve
+
+body:
+  - type: markdown
+    attributes:
+      value: ISSUES NOT FOLLOWING THIS TEMPLATE WILL BE CLOSED AND DELETED
+
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      options:
+        - label: I have read through the manual page (`man fzf`)
+          required: true
+        - label: I have searched through the existing issues
+          required: true
+
+  - type: input
+    attributes:
+      label: fzf version
+      placeholder: e.g. 0.48.1
```

Sometimes people don't realize they have a different version of fzf in their $PATH, so this will prevent such confusion.

```suggestion
      label: Output of `fzf --version`
      placeholder: e.g. 0.48.1 (d579e33)
```